### PR TITLE
Don't require f-strings when the formatee is a bytestring

### LINKED ISSUE #22: Should not trigger on byte strings
The following code triggers an FS001 error:
```py
a = 42
b"y=%d" % a
```
However, using an f-string with a byte string is a syntax error:
```pycon
>>> bf"y={a}"
```
```pytb
  File "<stdin>", line 1
    bf"y={a}"
      ^^^^^^^
SyntaxError: invalid syntax
```

(In this particular case `f"y={a}".encode("utf-8")` would work, but that's longer and doesn't always work correctly if there are byte literals in my string.)

### PR DESCRIPTION
**Pull request by akx**
Fixes https://github.com/MichaelKim0407/flake8-use-fstring/issues/22

### CODE REVIEW

File: flake8_use_fstring/base.py [hunk 1]
```diff
@@ -10,6 +12,18 @@
...
+
+
+def is_text_string_token(token: _TokenInfo) -> bool:
+    if not token.type == _token.STRING:
+        return False  # Not a string at all? Ignore.
+    # Get the prefix of the string (anything before the first quote)
+    prefix = prefix_re.match(token.string).group(0).lower()
+    if 'b' in prefix:
+        return False  # Smells like a bytestring. Ignore it.
+    return True
```
Comment (MichaelKim0407) [inline flake8_use_fstring/base.py:25]: It makes more sense to move these inside the class that's using them since they are not intended to be exposed.

**Comment by MichaelKim0407** [comment]
Thanks, looks good apart from the minor suggestion

File: flake8_use_fstring/base.py (hunk 1 above)
Comment (akx) [reply flake8_use_fstring/base.py]: I moved it outside the module altogether since it can (and should) be used by prefix.py too.

**Comment by MichaelKim0407** [comment]
Changes have been released. Thanks for your contribution!

**Pull request merged**